doc/examples/flare/lib/main.dart

```diff
@@ -0,0 +1,106 @@
+import 'dart:ui';
+import 'package:flutter/gestures.dart';
+import 'package:flame/flame.dart';
+import 'package:flame/game.dart';
+import 'package:flame/flare_animation.dart';
+import 'package:flame/components/flare_component.dart';
+import 'package:flame/text_config.dart';
+import 'package:flame/position.dart';
+
+import 'package:flutter/material.dart';
+
+void main() {
+  final game = MyGame();
+  runApp(game.widget);
+
+  Flame.util.addGestureRecognizer(TapGestureRecognizer()
+    ..onTapDown = (TapDownDetails evt) {
+      game.cycleAnimation();
+    });
+}
+
+class MyGame extends BaseGame {
+  final TextConfig fpsTextConfig = TextConfig(color: const Color(0xFFFFFFFF));
+
+  final paint = Paint()..color = Color(0xFFe5e5e5e5);
+  List<String> _animations = ["Stand", "Wave", "Jump", "Dance"];
+  int _currentAnimation = 0;
+
+  FlareAnimation flareAnimation;
+  bool loaded = false;
+
+  MyGame() {
+    _start();
+  }
+
+  @override
+  bool debugMode() => true;
+
+  void cycleAnimation() {
+    if (_currentAnimation == 3) {
+      _currentAnimation = 0;
+    } else {
+      _currentAnimation++;
+    }
+
+    flareAnimation.updateAnimation(_animations[_currentAnimation]);
+  }
+
+  void _start() async {
+    flareAnimation = await FlareAnimation.load("assets/Bob_Minion.flr");
+    flareAnimation.updateAnimation("Stand");
+
+    flareAnimation.x = 50;
+    flareAnimation.y = 50;
+
+    flareAnimation.width = 306;
+    flareAnimation.height = 228;
+
+    // Second animation
+    final flareAnimation2 = FlareComponent("assets/Bob_Minion.flr", "Wave", 306, 228);
+    flareAnimation2.x = 50;
+    flareAnimation2.y = 240;
+    add(flareAnimation2);
+
+    // Third animation
+    final flareAnimation3 = FlareComponent("assets/Bob_Minion.flr", "Jump", 306, 228);
+    flareAnimation3.x = 50;
+    flareAnimation3.y = 400;
+    add(flareAnimation3);
+
+    // Third animation
```

Comment:
third, third? hahaha
I think you can remove those comments about which animation is first or second, that is pretty much crystal clear to me